specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "2018-01-01",
+    "version": "2019-10-01",
```

Comment:
Please revert this to 2018-01-01 as this refers to the API version of EventGrid data plane, and 2018-01-01 is the last known GA version of EventGrid dataplane.